alfasim_sdk_api/detail/bootstrap_win.h

```diff
@@ -6,93 +6,88 @@
-        get_plugin_input_data_filepath_size = (get_plugin_input_data_filepath_size_func)GetProcAddress(this->dllHandle, "get_plugin_input_data_filepath_size");
-        get_plugin_input_data_reference = (get_plugin_input_data_reference_func)GetProcAddress(this->dllHandle, "get_plugin_input_data_reference");
-        get_plugin_variable = (get_plugin_variable_func)GetProcAddress(this->dllHandle, "get_plugin_variable");
-        get_field_id = (get_field_id_func)GetProcAddress(this->dllHandle, "get_field_id");
-        get_primary_field_id_of_phase = (get_primary_field_id_of_phase_func)GetProcAddress(this->dllHandle, "get_primary_field_id_of_phase");
-        get_phase_id = (get_phase_id_func)GetProcAddress(this->dllHandle, "get_phase_id");
-        get_layer_id = (get_layer_id_func)GetProcAddress(this->dllHandle, "get_layer_id");
-        get_state_variable_array = (get_state_variable_array_func)GetProcAddress(this->dllHandle, "get_state_variable_array");
-        get_simulation_array = (get_simulation_array_func)GetProcAddress(this->dllHandle, "get_simulation_array");
-        get_simulation_tracer_array = (get_simulation_tracer_array_func)GetProcAddress(this->dllHandle, "get_simulation_tracer_array");
-        get_simulation_quantity = (get_simulation_quantity_func)GetProcAddress(this->dllHandle, "get_simulation_quantity");
-        get_wall_interfaces_temperature = (get_wall_interfaces_temperature_func)GetProcAddress(this->dllHandle, "get_wall_interfaces_temperature");
-        get_flow_pattern = (get_flow_pattern_func)GetProcAddress(this->dllHandle, "get_flow_pattern");
-        get_plugin_input_data_table_quantity = (get_plugin_input_data_table_quantity_func)GetProcAddress(this->dllHandle, "get_plugin_input_data_table_quantity");
-        get_tracer_id = (get_tracer_id_func)GetProcAddress(this->dllHandle, "get_tracer_id");
-        get_tracer_name_size = (get_tracer_name_size_func)GetProcAddress(this->dllHandle, "get_tracer_name_size");
-        get_tracer_name = (get_tracer_name_func)GetProcAddress(this->dllHandle, "get_tracer_name");
-        get_tracer_ref_by_name = (get_tracer_ref_by_name_func)GetProcAddress(this->dllHandle, "get_tracer_ref_by_name");
-        get_tracer_partition_coefficient = (get_tracer_partition_coefficient_func)GetProcAddress(this->dllHandle, "get_tracer_partition_coefficient");
-        get_wall_layer_id = (get_wall_layer_id_func)GetProcAddress(this->dllHandle, "get_wall_layer_id");
-        set_wall_layer_property = (set_wall_layer_property_func)GetProcAddress(this->dllHandle, "set_wall_layer_property");
-        get_plugin_input_data_multiplereference_selected_size = (get_plugin_input_data_multiplereference_selected_size_func)GetProcAddress(this->dllHandle, "get_plugin_input_data_multiplereference_selected_size");
+    // Load shared object
+    WCHAR full_filepath[MAX_PATH];
+    full_filepath[0] = '\0';
```

Comment:
Why `0` index here?

Reply:
I'll use this to concatenate with other strings.
The initial value is "empty string" (Thus, `full_filepath[0] = '\0'`)